apps/cli/src/commands/send/erc20.ts

```diff
@@ -8,7 +8,7 @@ import {
     WalletClient,
 } from "viem";
 
-import { erc20PortalAbi, erc20PortalAddress } from "../../contracts.js";
+import { erc20PortalAbi, erc20PortalAddress, sunodoTokenAbi } from "../../contracts.js";
```

Comment:
You don't need to use a `sunodoTokenAbi`. Use the `erc20Abi` from `viem`.
The feature is valid for any ERC-20 token, not SunodoToken.